backend/src/services/db-service.ts

```diff
@@ -117,12 +127,17 @@ export class DbService {
   }
 
   async getImages(page: number, limit: number): Promise<string[]> {
-    const images = await Image.find({}).select(['path', 'date']).sort({date: -1}).limit(limit);
+    try {
+      const images = await Image.find({}).select(['path', 'date']).sort({date: -1}).limit(limit);
 
-    const sortedImages = this.sortImagesFromOldToNew(images);
-    const imagesPaths = this.retrieveImagesPaths(sortedImages);
+      const sortedImages = this.sortImagesFromOldToNew(images);
+      const imagesPaths = this.retrieveImagesPaths(sortedImages);
 
-    return this.getImagesPerPage(imagesPaths, page, PER_PAGE);
+      return this.getImagesPerPage(imagesPaths, page, PER_PAGE);
+    } catch (e) {
+      log.error(e);
```

Comment:
why do we need two types of logs here? and how can we find out in which function we got an error?